src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.scss

```diff
@@ -0,0 +1,17 @@
+@use './src/styles/vendor/variables' as vv;
+
+.indent {
+  display: table-cell;
+  padding-left: 17px;
+  padding-top: 8px;
+}
+
+.fa-info-circle {
+  color: vv.$info;
+  font-size: 12px;
+}
+
+.fa-exclamation-triangle {
+  color: vv.$danger;
+  font-size: 12px;
+}
```

Comment:
I'd try to avoid having per-component styles as much as possible: indent might be achieved with lists, and these 2 icons might be useful for other components, so better to define them at global scope.

Reply:
Adding the `ul` tag solved the indentation issue but added some weird margin space to the table cell. To overcome this and adding color to the icons, some style needs to be there in the .scss file, but its very minimal now.
![Screenshot from 2021-05-06 17-04-54](https://user-images.githubusercontent.com/66050535/117295899-0955ac80-ae92-11eb-9062-4e253e608a89.png)